### PR TITLE
Update MxBuild documentation

### PR DESCRIPTION
**Pull request by ghost**
Include links to different flavors of MxBuild for Windows OS depending on a version

### CODE REVIEW

File: content/en/docs/refguide/general/mxbuild.md [hunk 1]
```diff
@@ -11,7 +11,16 @@ tags: ["build", "deploy", "deployment package", "command-line", "studio pro"]
...
+The version of MxBuild which you need is dependent on the version of the Mendix model you want to build, and on the host operating system you want to use.
+
+Please refer to the following table to find your correct MxBuild by entering a URL from a correct row into a browser and replacing `mxversion` with your own, full Mendix version number:
+
+| Operating system | Mendix version    | URL |
+| ---------------- | ----------------- | --- |
+| Linux (x64)      | All versions      | `https://cdn.mendix.com/runtime/mxbuild-{mxversion}.tar.gz`      |
+| Windows (x64)    | < 9.20            | `https://cdn.mendix.com/runtime/mxbuild-{mxversion}.tar.gz`      |
+| Windows (x64)    | 9.20 - 9.22       | `https://cdn.mendix.com/runtime/mono-mxbuild-{mxversion}.tar.gz` |
+| Windows (x64)    | >= 9.23           | `https://cdn.mendix.com/runtime/win-mxbuild-{mxversion}.tar.gz`  |
```
Comment (ghost) [inline content/en/docs/refguide/general/mxbuild.md]: There is a way to combine these rows with some nice HTML but I'm not sure if we want it?

<table>
    <thead>
        <tr>
            <th>Operating system</th>
            <th>Mendix version</th>
            <th>URL</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>Linux (x64)</td>
            <td>All versions</td>
            <td><code>https://cdn.mendix.com/runtime/mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
        <tr>
            <td rowspan=3>Windows (x64)</td>
            <td>&lt; 9.20</td>
            <td><code>https://cdn.mendix.com/runtime/mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
        <tr>
            <td>9.20 - 9.22</td>
            <td><code>https://cdn.mendix.com/runtime/mono-mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
        <tr>
            <td>&gt;= 9.23</td>
            <td><code>https://cdn.mendix.com/runtime/win-mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
    </tbody>
</table>

Comment (MariaShaposhnikova) [reply content/en/docs/refguide/general/mxbuild.md]: @aov-mx , I can use a break line to divide them within one column.

Comment (MariaShaposhnikova) [reply content/en/docs/refguide/general/mxbuild.md]: @aov-mx , I tested my suggestion on the local build, but it does not look nice, I have changed it back to different lines, looks okay to me.

Comment (aov-mx) [reply content/en/docs/refguide/general/mxbuild.md]: If our Docs are okay with HTML, here's the code:

```html
<table>
    <thead>
        <tr>
            <th>Operating system</th>
            <th>Mendix version</th>
            <th>URL</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>Linux (x64)</td>
            <td>All versions</td>
            <td><code>https://cdn.mendix.com/runtime/mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
        <tr>
            <td rowspan=3>Windows (x64)</td>
            <td>&lt; 9.20</td>
            <td><code>https://cdn.mendix.com/runtime/mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
        <tr>
            <td>9.20 - 9.22</td>
            <td><code>https://cdn.mendix.com/runtime/mono-mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
        <tr>
            <td>&gt;= 9.23</td>
            <td><code>https://cdn.mendix.com/runtime/win-mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
    </tbody>
</table>
```

Comment (ghost) [reply content/en/docs/refguide/general/mxbuild.md]: @MariaShaposhnikova If our Docs are okay with HTML, here's the code:

```html
<table>
    <thead>
        <tr>
            <th>Operating system</th>
            <th>Mendix version</th>
            <th>URL</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>Linux (x64)</td>
            <td>All versions</td>
            <td><code>https://cdn.mendix.com/runtime/mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
        <tr>
            <td rowspan=3>Windows (x64)</td>
            <td>&lt; 9.20</td>
            <td><code>https://cdn.mendix.com/runtime/mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
        <tr>
            <td>9.20 - 9.22</td>
            <td><code>https://cdn.mendix.com/runtime/mono-mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
        <tr>
            <td>&gt;= 9.23</td>
            <td><code>https://cdn.mendix.com/runtime/win-mxbuild-{mxversion}.tar.gz</code></td>
        </tr>
    </tbody>
</table>
```

Comment (MariaShaposhnikova) [reply content/en/docs/refguide/general/mxbuild.md]: @aov-mx , sorry, I saw it after I merged. We are okay with occasional html, yet we try avoiding it where possible for maintanance reasons. I think that the default version looks okay, I will keep it.

**Pull request merged**